src/shader/camera.rs

```diff
@@ -17,41 +37,26 @@ impl From<config::PresetCamera> for Camera {
         match cam {
             Perspective(persp) => persp.into(),
             //TODO(#4): This should be implemented as part of #4.
-            Custom {position, target} => unimplemented!(),
+            Custom(camera) => camera.into(),
         }
     }
 }
 
 impl From<config::Perspective> for Camera {
     fn from(persp: config::Perspective) -> Self {
-        //TODO(#4): This should be reimplemented properly as part of #4. These placeholder values
-        // are only meant to work for the desired angles of bigboi. The angles are slightly tilted.
-        // In the actual implementation they should be straight on.
+
+        // NOTE: PerspectiveLeft means point the camera to the left side of the x axis( negative x values ).
+        // I.e: the camera is actually looking at the right side of the model
         use config::Perspective::*;
-        let view = match persp {
-            PerspectiveFront => Mat4::rotation_x(PI/8.0) * Mat4::rotation_y(0.0*PI/2.0),
-            PerspectiveBack => unimplemented!("TODO"),
-            PerspectiveLeft => unimplemented!("TODO"),
-            PerspectiveRight => Mat4::rotation_x(PI/8.0) * Mat4::rotation_y(-1.0*PI/2.0),
-            PerspectiveTop => unimplemented!("TODO"),
-            PerspectiveBottom => unimplemented!("TODO"),
+        let mut eye = match persp {
+            PerspectiveFront => Vec4::new(0.0,1.0,8.5,1.0),
```

Comment:
Do we want to have the camera directly in front (y = 0.0) or slightly above like we have here (y = 1.0)? I think we should go for directly in front since that's probably more intuitive. (It's hard to pick a y value that would work for all models anyway so we probably should just stick with y = 0.0.)